src/Http/Authentication.Abstractions/src/AuthenticateResult.cs

```diff
@@ -89,7 +91,7 @@ public static AuthenticateResult Success(AuthenticationTicket ticket)
     /// <returns>The result.</returns>
     public static AuthenticateResult NoResult()
     {
-        return new AuthenticateResult() { None = true };
+        return _noResult;
```

Comment:
Clone returning the same is unexpected by maybe fine since you can't do much with it anyways?